doc/sailfish-webview-popups-popupprovider.qdoc

```diff
@@ -122,3 +122,11 @@
   The component specified must implement the
   \l{BlockedTabPopupInterface}{blocked tab popup} interface.
 */
+
+/*!
+  \qmlproperty var PopupProvider::selectorPopup
+  \brief The selector popup component descriptor
+
+  The component specified must implement the
+  \l{SelectorPopupInterface}{selector popup} interface.
```

Comment:
As commented, there's SingleSelectPage and MultiSelectDialog under pickers.

Reply:
Similarly, I'll resolve this following the discussion on the related changes in embedlite-components: https://github.com/sailfishos/embedlite-components/pull/5#discussion_r667088133